src/sql/workbench/contrib/query/browser/queryResultsView.ts

```diff
@@ -102,7 +102,7 @@ class ResultsView extends Disposable implements IPanelView {
 	}
 
 	onShow(): void {
-		if (this._state) {
+		if (this._state && this._runner) {
```

Comment:
It seems odd to me that these are so tied together. It's probably not valid for them to not be set but I'd generally default to having them set themselves separately and if one is undefined for some reason then we should handle/error appropriately.